website/app/templates/patterns.hbs

```diff
@@ -16,8 +16,8 @@
     </p>
     <h2 class="doc-text-h2">Creating your own pattern?</h2>
     <p class="doc-text-body">
-      <LinkTo class="doc-link-generic" @route="show" @model="about/support">Contact the Design Systems Team</LinkTo>
-      for best practices and suggestions when building and defining local components and patterns.
+      <LinkTo class="doc-link-generic" @route="show" @model="about/support">Contact the Helios Design System team</LinkTo>
```

Comment:
It's a bit inconsistent in this PR, some of the team references are "Systems" plural and some are "System" singular. I think in the writing guidelines we recommend plural (but @heatherlarsen check me on that). Also treating the team as a proper noun  would mean the capitalization of "Team"

[Reference](https://docs.google.com/document/d/1WyoJVpWFVgWbCnZ28WW0gRJwJlj-qV5oY0ExhDR2obs/edit#bookmark=id.56sitq2d1p5h)

```suggestion
      <LinkTo class="doc-link-generic" @route="show" @model="about/support">Contact the Helios Design Systems Team</LinkTo>
```